release-notes/opensearch-security-dashboards-plugin.release-notes-1.0.0.0-beta1.md

```diff
@@ -1,18 +1,25 @@
 ## Version 1.0.0.0-beta1
 
 ### Enhancements
-* Migrates plugin name from `Opendistro` to `Open Search`
```

Comment:
Why are we removing items? Were they added unexpectedly?

Reply:
there is a PR for that: name change to opensearch dashboard #743